README.md

```diff
@@ -13,7 +13,7 @@
 [![Build Docker Image](https://github.com/drifting-in-space/plane/actions/workflows/plane2-build-image.yml/badge.svg)](https://github.com/drifting-in-space/plane/actions/workflows/plane2-build-image.yml)
 [![Tests](https://github.com/drifting-in-space/plane/actions/workflows/plane2-tests.yml/badge.svg)](https://github.com/drifting-in-space/plane/actions/workflows/plane2-tests.yml)
 
-Plane is infrastructure software for **running stateful WebSocket backends at scale**. Plane is heavily inspired by [Figma’s mulitplayer infrastructure](https://www.figma.com/blog/rust-in-production-at-figma/), which dynamically spawns a process for each active document.
+Plane is a distributed system for **running stateful WebSocket backends at scale**. Plane is heavily inspired by [Figma’s mulitplayer infrastructure](https://www.figma.com/blog/rust-in-production-at-figma/), which dynamically spawns a process for each active document.
 
 Use cases include:
```

Comment:
i think we can expand on this section to include a list of features and then use cases ie https://docs.vllm.ai/en/latest/#

having a features section would help cement how plane as websocket backend is unique. some features that could be useful might explain how the backend is scaled

Reply:
I'll punt this to a future PR, but if you have specific suggestions on what to add feel free to open another PR when this is merged.